include/re_sa.h

```diff
@@ -6,10 +6,18 @@
 #if defined(WIN32)
 #include <winsock2.h>
 #include <ws2tcpip.h>
+#if !defined(UNIX_PATH_MAX)
+#define UNIX_PATH_MAX 108
+typedef struct sockaddr_un {
+	ADDRESS_FAMILY sun_family;
+	char sun_path[UNIX_PATH_MAX];
+} SOCKADDR_UN, *PSOCKADDR_UN;
+#endif
```

Comment:
does unixsock work on windows ?

Reply:
Testing is on my list, but its available (including afunix.h does not work for mingw and curl does the same fallback):

https://devblogs.microsoft.com/commandline/af_unix-comes-to-windows/